boards/feather-m0/include/arduino_pinmap.h

```diff
@@ -61,6 +56,38 @@ extern "C" {
 #define ARDUINO_PIN_23          GPIO_PIN(PB, 10) /**< D23 (SPI MOSI) */
 #define ARDUINO_PIN_24          GPIO_PIN(PB, 11) /**< D24 (SPI SCK) */
 
+#if defined(BOARD_FEATHER_M0_WIFI)
+#define ARDUINO_PIN_2           GPIO_PIN(PA, 14) /**< D2 (WINC_CHIP_EN) */
+#define ARDUINO_PIN_3           GPIO_UNDEF       /**< D3 */
+#define ARDUINO_PIN_4           GPIO_PIN(PA, 8)  /**< D4 (WINC_RST) */
+#define ARDUINO_PIN_7           GPIO_PIN(PA, 21) /**< D7 (WINC_IRQ) */
+#define ARDUINO_PIN_8           GPIO_PIN(PA, 6)  /**< D8 (WINC_CS) */
+#elif defined(BOARD_FEATHER_M0_LORA)
+#define ARDUINO_PIN_2           GPIO_UNDEF       /**< D2 */
+#define ARDUINO_PIN_3           GPIO_PIN(PA, 9)  /**< D3 (LORA_IRQ) */
+#define ARDUINO_PIN_4           GPIO_PIN(PA, 8)  /**< D4 (LORA_RST) */
+#define ARDUINO_PIN_7           GPIO_UNDEF       /**< D7 */
+#define ARDUINO_PIN_8           GPIO_PIN(PA, 6)  /**< D8 (LORA_CS) */
+#elif defined(BOARD_FEATHER_M0_BLE)
+#define ARDUINO_PIN_2           GPIO_UNDEF       /**< D2 */
+#define ARDUINO_PIN_3           GPIO_UNDEF       /**< D3 */
+#define ARDUINO_PIN_4           GPIO_PIN(PA, 8)  /**< D4 (BLE_SWDIO/RST) */
+#define ARDUINO_PIN_7           GPIO_PIN(PA, 21) /**< D7 (BLE_IRQ) */
+#define ARDUINO_PIN_8           GPIO_PIN(PA, 6)  /**< D8 (BLE_CS) */
+#elif defined(BOARD_FEATHER_M0_LOGGER)
```

Comment:
You should define this as a board. See my comment. Then, the conditional would be valid. But without having a board definition for it ...

Reply:
Ok, removed the blocks for BLE and Adalogger.
Will add feather-m0-adalogger in a later PR.